cloud-aws-common/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/common/AwsPlatformResourcesTest.java

```diff
@@ -317,6 +317,40 @@ public void collectEncryptionKeysWhenWeGetBackInfoThenItShouldReturnListWithElem
         assertEquals(4L, cloudEncryptionKeys.getCloudEncryptionKeys().size());
     }
 
+    @Test
+    public void collectEncryptionKeysWhenWeGetBackInfoThenItShouldReturnListWithPaginatedElements() {
+        ListKeysResult listKeysResult = new ListKeysResult();
+
+        Set<KeyListEntry> listEntries = new HashSet<>();
+        for (int i = 0; i < 52; i++) {
```

Comment:
Is it possible not to hard code this as 52 and make this configurable as a parameter? This way, you can do a unit test with multiple pages of results, like for example 100 plus.